zhaquirks/sinope/thermostat.py

```diff
@@ -164,3 +164,59 @@ class SinopeTH1400ZB(SinopeTechnologiesThermostat):
             }
         }
     }
+
+
+class SinopeTH1300ZB(SinopeTechnologiesThermostat):
+    """TH1300ZB thermostat."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=769 device_version=1
+        # input_clusters=[0, 3, 4, 5, 513, 516, 1026, 1794, 2820, 2821, 65281]
+        # output_clusters=[10, 25, 65281]>
+        MODELS_INFO: [(SINOPE, "TH11300ZB")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha_p.PROFILE_ID,
+                DEVICE_TYPE: zha_p.DeviceType.THERMOSTAT,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    Identify.cluster_id,
+                    Groups.cluster_id,
+                    Scenes.cluster_id,
+                    Thermostat.cluster_id,
+                    UserInterface.cluster_id,
+                    TemperatureMeasurement.cluster_id,
+                    Metering.cluster_id,
+                    ElectricalMeasurement.cluster_id,
+                    Diagnostic.cluster_id,
+                    SINOPE_MANUFACTURER_CLUSTER_ID,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Time.cluster_id,
+                    Ota.cluster_id,
+                    SINOPE_MANUFACTURER_CLUSTER_ID,
+                ],
+            }
+        },
+    }
+
+    replacement = {
+        ENDPOINTS: {
+            1: {
+                INPUT_CLUSTERS: [
+                    Basic,
```

Comment:
@Adminiuga We have been using the id's when not replacing the clusters with different implementations. Curious on your thoughts for this? Should we use the id's or the clusters?

Reply:
Yep, should be using Cluster.ckuster_id here and use custom clusters for thermostat and manufacturer specific one